src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -402,7 +404,8 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<AndroidCustomMetaDataForReferences Include="@(_AndroidAssemblySkipCases)" />
 </ItemGroup>
 
-<Target Name="_AddAndroidCustomMetaData">
+<Target Name="_AddAndroidCustomMetaData"
+    DependsOnTargets="_ConvertAndroidMamMappingFileToXml;_CollectAndroidRemapMembers">
```

Comment:
I *think* (and @jonathanpeppers can correct me :) ) we tend to prefer that we explicitly define the dependencies in
`src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.BuildOrder.targets` for .NET and `src/Xamarin.Android.Build.Tasks/Xamarin.Android.Legacy.targets` for Legacy. We can do this in addition to `DependsOnTargets` but it does help when figuring out the build order if we have it in the other files too.

This is for .NET 7+ only right?

Reply:
Correct, this is .NET 7+ only.